SimpleLottery/Model/Lottery.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Lottery.swift
+//  SimpleLottery
+//
+//  Created by HyungJung Kim on 18/03/2019.
+//  Copyright © 2019 HyungJung Kim. All rights reserved.
+//
+
+import Foundation
+
+struct Lottery {
+    
+    static let price = 1000
+    static let numbersRange = Range(1 ... 45)
+    static let numbersMaximumCount = 6
+    
+    let numbers: [Int]
+    
+    init() {
+        self.numbers = Lottery.generatedNumbers()
+    }
+    
+    init(numbers: [Int]) {
+        self.numbers = numbers
+    }
+    
+    private static func generatedNumbers() -> [Int] {
```

Comment:
```suggestion
    private static func generateNumbers() -> [Int] {
```

`generatedNumbers` 생성된 숫자 (명사) 보다 `generateNumber` 숫자 생성 (동사)가 좋지 않을까요?

Reply:
side effect 없는 메서드이기 때문에 명사구로 읽혀야 할 것 같습니다.